packages/core/src/shared/vscode/commands2.ts

```diff
@@ -622,13 +622,15 @@ export const defaultTelemetryThrottleMs = 300_000 // 5 minutes
 async function runCommand<T extends Callback>(fn: T, info: CommandInfo<T>): Promise<ReturnType<T> | void> {
     const { id, args, label, logging, compositeKey } = { logging: true, ...info }
     const logger = logging ? getLogger() : new NullLogger()
-    const withArgs = args.length > 0 ? ` with arguments [${args.map(String).join(', ')}]` : ''
     const threshold = info.telemetryThrottleMs ?? defaultTelemetryThrottleMs
     const instrumenter = logging
         ? getInstrumenter({ id, args, compositeKey: compositeKey ?? {} }, threshold || 0, info.telemetryName)
         : undefined
 
-    logger.debug(`command: running ${label}${withArgs}`)
+    logger.debug(
+        `command: running ${label} with arguments: %O`,
```

Comment:
question: is there somewhere that shows the available formatting symbols (%s, %O)?

Reply:
It's part of `console.log` and node's `util.inspect()`. This seems like the best reference: https://console.spec.whatwg.org/#formatting-specifiers